pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/v2/PNConfiguration.kt

```diff
@@ -248,7 +256,11 @@ interface PNConfiguration : com.pubnub.api.v2.PNConfiguration {
          *
          * @see [HttpLoggingInterceptor]
          */
-        val httpLoggingInterceptor: HttpLoggingInterceptor?
+        @Deprecated(
+            message = "This setting is deprecated. Use customLoggers  instead",
+            level = DeprecationLevel.WARNING
+        )
+        val httpLoggingInterceptor: HttpLoggingInterceptor? // todo deprecate
```

Comment:
Perhaps ``// todo deprecate`` comment is no longer required

Reply:
Of course. Good catch. I will fix that.